pkg/http/controller.go

```diff
@@ -8,11 +8,15 @@ import (
 )
 
 func (s *HTTPServer) CreateNetwork(c *gin.Context) {
-	result := s.apis.CreateNetwork(&model.Network{
-		ContractAddressURL: "aa.com",
-		StateDumpURL:       "bb.com",
+	result, err := s.apis.CreateNetwork(&model.Network{
```

Comment:
We can set the argument of apis.CreateNetwork based on user's request later. 😄

Reply:
I will update it later!